pygyro/advection/test_advection.py

```diff
@@ -624,7 +624,7 @@ def test_Phi_deriv_dz(phiOrder, zOrder):
 
         err = approxGrad-exactGrad
 
-        l2[i] = np.sqrt(np.trapz(np.trapz(err**2, dx=dz), dx=dtheta))
+        l2[i] = np.sqrt(np.trapezoid(np.trapezoid(err**2, dx=dz), dx=dtheta))
```

Comment:
Are you using `trapezoid` from SciPy or NumPy?

Reply:
Ah this might be related to me renaming the `master` branch to `main`

Reply:
I was asking because in the same file you also import it from SciPy, if I am not mistaken...

Reply:
Ah, I assumed it was because I wrote SciPy in the PR description

Reply:
You are right though we are using both